pkg/components/common/common.go

```diff
@@ -282,6 +290,107 @@ func (o *Operator) Uninstall(
 	return nil
 }
 
+func (o *Operator) DownloadKind(ctx context.Context) error {
+	return o.executor.DownloadKind()
+}
+
+func (o *Operator) CreateKindCluster(ctx context.Context) error {
+	cmd := "kind create cluster"
```

Comment:
Can we add the `--name` and `--image` options?
Like `kind create cluster --name openfunction --image kindest/node:v1.20.0`
For OpenFunction v0.4.0, maybe we can specify the Kubernetes of v1.20.0 by default?